README.dev.md

```diff
@@ -92,6 +95,17 @@ virtualenv -p python .venv
 .venv\Scripts\activate.ps1
 python -m pip install -r requirements.txt
 ```
+
+#### Conda virtual environment
+To setup RP2 in a conda environment, enter the following commands:
+```
+cd <rp2_directory>
+conda create -n <env_name>
```

Comment:
Are you missing instructions on conda installation? Not sure about Windows, but I tried it on Mac and I got "conda: command not found"